activesupport/lib/active_support/claims.rb

```diff
@@ -0,0 +1,63 @@
+require 'active_support/time'
+
+module ActiveSupport
+  # +Claims+ supports <tt>ActiveSupport::MessageVerifier</tt> by
+  # representation of message in the form JWT Claims and verifying
+  # the claims of the signed message. 
+  class Claims
+    class InvalidClaims < StandardError; end
+    class ExpiredClaims < StandardError; end
+    
+    attr_reader :payload, :purpose, :expires_at
+    DEFAULT_PURPOSE = 'universal'
+
+    # Keyword argument +:payload+ is mandatory
```

Comment:
Because it'll raise if people don't pass a payload argument:

```irb
irb(main):001:0> class Claims
irb(main):002:1>   def initialize(payload:, **options)
irb(main):003:2>   end
irb(main):004:1> end
=> :initialize
irb(main):005:0> Claims.new
ArgumentError: missing keyword: payload
	from (irb):2:in `initialize'
	from (irb):5:in `new'
	from (irb):5
	from /Users/kasperhansen/.rbenv/versions/2.2.2/bin/irb:11:in `<main>'
```